### DISCUSSION TITLE
Spam score 0 from mail tester

### DISCUSSION THREAD

**Discussion by mredodos** [postalserver/postal#2667] [Help with using Postal] [answered]
![image](https://github.com/postalserver/postal/assets/42175654/23ebcbba-3ae7-42e4-9480-7516283041d2)

![image](https://github.com/postalserver/postal/assets/42175654/28ac7dc5-5b26-4be1-99c5-21bd2ea95b2a)
![image](https://github.com/postalserver/postal/assets/42175654/8afe0b07-9c28-4fc6-ba76-503017aae68b)
![image](https://github.com/postalserver/postal/assets/42175654/c5da1705-10ba-4976-aac4-1c0a2b77ca78)

i dont understand what im missing

**Comment by sinnbeck** [comment]
Looks like you set up bearswissmail.ch but try to send from bsmrp.bearswissmail.ch which is a different domain (a sub domain)?

**Reply by sinnbeck** [reply]
Just noticed that google does not have any records for that domain. Maybe digital ocean is having replication issues with dns

https://toolbox.googleapps.com/apps/dig/#SOA/

**Reply by mredodos** [reply]
bsmrp.bearswissmail.ch is the return path of postal

**Reply by sinnbeck** [reply]
Ah ok. Thought that was normally psrp.senderdomain.com and rp.postal.domain.com :) Ignore that then

Anyways. Google still has no knowledge of your domain at all. And mail-tester uses their dns for lookups as far as I know (8.8.8.8)

**Reply by mredodos** [reply]
http://leafdns.com/index.cgi?testid=8A8BE285 it seems all ok, i dont know  why google dont recognize the domain

**Reply by sinnbeck** [reply]
Yeah mxtoolbox also has no problems with it. But if mail-tester.com uses google DNS it will give you a 0 score as it cannot find a single record. I would try contacting digitalocean and ask why google dns returns nothing for the domain.

**Reply by willpower232** [reply]
Yeah something weird is going on https://www.whatsmydns.net/#MX/bearswissmail.ch

**Reply by mredodos** [reply]
i dont understand this iybesc@bsmrp.bearswissmail.ch where come from that email?

**Reply by willpower232** [reply]
thats the return path so if the email bounces, the other email server will send an email to that address which goes to postal so it knows the message bounced

**Reply by mredodos** [reply, marked as answer]
ok after remove NS and add it again it seems works, now i have 9,5

**Comment by willpower232** [comment]
Postal gives you green ticks when the DNS matches what is set in the config file but that doesn't mean there are no errors unfortunately.

I presume that is the return path of the email to deal with bounces, it can also be used to verify that the email came from a legitimate source and that source can be updated about the status of the email.

Are you able to share what DNS records you do have set up?

**Reply by mredodos** [reply]
yes the problem it seems related to return path

![image](https://github.com/postalserver/postal/assets/42175654/e78f042d-07c4-4617-bbe1-2694d483a9f6)

Actually in that config missing the CNAME i have try to remove it for test if something change 

but the issue trigger with that dns in the config
